src/stories/src/components/split-view/split-view.mdx

```diff
@@ -0,0 +1,265 @@
+
+</PropertyDef>
+
+<PropertyDef name="min" type="number" defaultValue="0">
+
+The minimum allowed size in pixels.
+
+</PropertyDef>
+
+<PropertyDef name="max" type="number | undefined" defaultValue="undefined">
+
+The maximum allowed size in pixels.
+
+</PropertyDef>
+
+<PropertyDef name="open" type="boolean" defaultValue="true">
+
+Controls whether the panel is displayed.
+
+> **Important:** there is no UI built into the component to open closed panels. If you allow the
+> user to close panels you should also provide a mechanism to reopen them.
+
+</PropertyDef>
+
+<PropertyDef name="label" type="string" defaultValue="'Split view panel'">
+
+Sets the accesible label for the resize handle.
+
+> **Important:** for strong accessibility, a unique value should be provided that describes the
+> content or purpose of the panel.
+
+</PropertyDef>
+
+<PropertyDef name="disabled" type="boolean" defaultValue="false">
+
+Sets whether resize handle interactions are disabled.
+
+</PropertyDef>
+
+<PropertyDef name="autoClose" type="boolean" defaultValue="false">
+
+Sets whether the panel closes when its size reaches zero.
+
+</PropertyDef>
+
+<PropertyDef name="disableClose" type="boolean" defaultValue="false">
+
+Sets whether the user can close the panel.
+
+</PropertyDef>
+
+</PageSection>
+
+<PageSection>
+
+## Events
+
+| Name                                  | Description
+| :------------------------------------ | ----------------
+| `forge-split-view-panel-resize-start` | Emits when the resize handle is grabbed.
+| `forge-split-view-panel-resize-end`   | Emits when the resize handle is released.
+| `forge-split-view-panel-resize`       | Emits when the split view panel is resized with the panel's size in pixels along the orientation axis.
+| `forge-split-view-panel-did-open`     | Emits after the split view panel opens.
+| `forge-split-view-panel-did-close`    | Emits after the split view panel closes.
+
+</PageSection>
+
+<PageSection>
+
+## Keyboard shortcuts
+
+| Name                                | Description
+| :-----------------------------------| :----------------
+| `arrow left`                        | Resizes the panel to the left if `orientation` is horizontal.
+| `arrow right`                       | Resizes the panel to the right if `orientation` is horizontal.
+| `arrow up`                          | Resizes the panel up if `orientation` is vertical.
+| `arrow down`                        | Resizes the panel down if `orientation` is vertical.
+| `enter`                             | Closes the panel.
+| `home`                              | Resizes the panel to its minimum size.
+| `end`                               | Resizes the panel to its maximum size.
+
+> An arrow key combined with `shift` resizes faster, ten pixels at a time. On its own an arrow key
+> resizes one pixel at a time.
+
+> The resize handle is focusable and receives all keyboard events.
+
+</PageSection>
+
+<PageSection>
```

Comment:
Let's add the `CSS parts` information here. See docs pages for other components as an example.